ui/page/staking/overview.go

```diff
@@ -348,7 +348,7 @@ func (pg *Page) stakeLiveSection(gtx layout.Context) layout.Dimensions {
 			}),
 			layout.Rigid(func(gtx C) D {
 				if len(pg.liveTickets) == 0 {
-					noLiveStake := pg.Theme.Label(values.TextSize16, "No live tickets yet.")
+					noLiveStake := pg.Theme.Label(values.TextSize16, "No current tickets.")
```

Comment:
i kind of have my reservation on the word `current` used how about `No active tickets` or `No ticket found`